prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -1037,3 +1037,39 @@ periodics:
         requests:
           memory: 200Mi
           cpu: 80m
+
+- name: kyma-gke-compass-integration-periodic
+  cron: "00 00 * * *" # "At 00:00 every day"
+  skip_report: false
+  decorate: true
+  path_alias: github.com/kyma-project/kyma
```

Comment:
Why do you define `path_alias`? Isn't it enough to have `extra_refs` defined?

Reply:
`path_alias` removed